lib/Onyx.js

```diff
@@ -428,17 +460,15 @@ function set(key, val) {
  * @returns {Promise}
  */
 function multiSet(data) {
-    // AsyncStorage expenses the data in an array like:
-    // [["@MyApp_user", "value_1"], ["@MyApp_key", "value_2"]]
-    // This method will transform the params from a better JSON format like:
-    // {'@MyApp_user': 'myUserValue', '@MyApp_key': 'myKeyValue'}
-    const keyValuePairs = _.reduce(data, (finalArray, val, key) => ([
-        ...finalArray,
-        [key, JSON.stringify(val)],
-    ]), []);
+    const keyValuePairs = prepareKeyValuePairsForStorage(data);
+
+    // Capture (non-stringified) changes to cache
+    cache.merge(_.pairs(data));
```

Comment:
Isn't this basically doing `multiMerge` not `multiSet()`?

```js
// previous value {loading: true}
Onyx.multiSet(['someKey', {data: 'test'}]); // -> {loading: true, data: 'test'}
```

and the value should be `{data: 'test'}` ?